multegula.go

```diff
@@ -501,14 +587,39 @@ func main() {
 	gameType := uiGetGameType()
 
 	if gameType == defs.GAME_TYPE_MULTI {
-		// get fellow players
+
+		var peers *[]messagePasser.Node
+		var err error
+		var lastTime int64
+		//Define ourselves
 		localNode := messagePasser.Node{Name: localNodeName, IP: "127.0.0.1", Port: *gamePortFlag}
-		peers, err := bootstrapClient.GetNodes(localNode)
-		if err != nil {
-			fmt.Println("Couldn't get peers:", err)
-			panic(err)
+
+		//Check to see if state file exists, and if it does, read timestamp from it.
+		//If timestamp is too old, delete and ignore.  If not, read nodes and ignore bootstrap.
+		lastTime,err = LoadTime("lastgametime.mlg")
+		if (err != nil) && (time.Now().Unix() - int64(300) > int64(lastTime)) {
```

Comment:
Yes.  You're right.  We want to run normal bootstrapping if the file does not exist OR the timestamp is old.